testplan/web_ui/testing/src/Report/__tests__/InteractiveReport.test.js

```diff
@@ -8,6 +8,68 @@ import InteractiveReport from '../InteractiveReport.js';
 import {FakeInteractiveReport} from '../../Common/sampleReports.js';
 import {ReportToNavEntry} from '../../Common/utils.js';
 
+const INITIAL_REPORT = {
+  "uid": "Assertions Example",
+  "timer": {},
+  "status": "ready",
+  "meta": {},
+  "status_override": null,
+  "attachments": {},
+  "tags_index": {},
+  "name": "Assertions Example",
+  "parent_uids": [],
+  "type": "TestGroupReport",
+  "category": "testplan",
+  "entries": [{
+    "uid": "Assertions Test",
```

Comment:
uid != name. It's not important, but maybe uid="AssertionsTestUID" would be better to avoid mishandling in the test.